exts/cesium.omniverse/cesium/omniverse/extension.py

```diff
@@ -407,3 +413,18 @@ def show_debug_window(self, _menu, value):
             asyncio.ensure_future(self._dock_window_async(self._debug_window))
         elif self._debug_window is not None:
             self._debug_window.visible = False
+
+    def _setup_credits_viewport_frames(self):
+        self._destroy_credits_viewport_frames()
+        viewport_frames = []
+        for instance in get_viewport_window_instances():
+            credits_viewport_frame = CesiumCreditsViewportFrame(_cesium_omniverse_interface, instance)
+            viewport_frames.append(credits_viewport_frame)
+        self._credits_viewport_frames = viewport_frames
+
+    def _destroy_credits_viewport_frames(self):
+        for credits_viewport_frame in self._credits_viewport_frames:
+            if credits_viewport_frame is not None:
+                credits_viewport_frame.destroy()
+                credits_viewport_frame = None
+        CreditsViewportController().clear_handlers()
```

Comment:
@weegeekps Should clear_handlers have a classmethod decorator so the call is `CreditsViewportController.clear_handlers()`?

Reply:
Probably not given the move to making it less a pure singleton, but let me mull over it some and get back to you in the morning.

Reply:
@weegeekps The controller's been refactored based on our discussion about not using a singleton.